src/loaders.ts

```diff
@@ -174,19 +174,21 @@ export const defaultSignatureLookup: SignatureLookup = new MultiSignatureLookup(
 
 type APIKeys = {
   ETHERSCAN_API_KEY?: string,
+  ETHERSCAN_BASE_URL?: string,
+  SOURCIFY_CHAIN_ID?: number,
```

Comment:
Would be better as a string so that we can use things like `process.env` where everything is a string? (Or maybe `string|number`)